Langs/README.md

```diff
@@ -0,0 +1,38 @@
+# Guidelines
+
+## Requirements
+
+- **Find Your Language Code Here:** https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm
+- Always use the latest template (0000.lang)
+- Use hex code(number) of your language.
+- Please use your GitHub username as Translator in MetaData section.
+- All strings must be filled out to be accepted.
+- Due to space restrictions, translations do not have to be 1:1.
+- Please do not use slang, due to it being hard to judge/approve.
+- File must be UTF-8 encoded, use GitHub editor and it should take care of that.
+
+## Testing Translations Locally
+1. Go to folder: *%localappdata%\WhyNotWin11\langs*
```

Comment:
instead of 1. 2. 3. you can use 1. 1. 1. so you won't have to change these numbers each time.

Reply:
Rewrite lists with 1. 1.